src/applications/simple-forms/21-0845/sass/0845-auth-disclose.scss

```diff
@@ -6,17 +6,87 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
 
+$font-family-serif: "Bitter", "Georgia", "Cambria", "Times New Roman", "Times",
```

Comment:
I understand adding all these CSS adjustments will make the form look just right. 👍
From the standpoint of tooling, it's also in our best interest to make this file as small as possible, so that bugs with css can be prominently visible so that we can be aware of the CSS issues to fix for every form rather than individual forms. For the RJSF forms it's fine, but If you make a form using the new web components/patterns, can you try to not add all these adjustments? Because we'd like to fix it in our new patterns and/or shared CSS / core CSS rather than with custom CSS.